src/wanna/core/services/pipeline.py

```diff
@@ -187,6 +187,7 @@ def _export_pipeline_params(
                 if pipeline_instance.service_account
                 else self.config.gcp_profile.service_account
             ),
+            "encryption_spec_key_name": encryption_spec_key_name,
```

Comment:
I think this should only be added to dict if exists(see tfboard below), instead of adding a "None" to the environment.

thx

Reply:
Ok done